backend/eventloop/db_reconciler.go

```diff
@@ -190,43 +191,47 @@ func cleanOrphanedEntriesfromTable_DTAM_DeleteEntry(ctx context.Context, deplToA
 		}
 	}
 
-	log := logger.WithValues("applicationID", deplToAppMapping.Application_id)
-
```

Comment:
Since this function is also working in SyncOperations and DeplToAppMapping tables, I thought of giving table type in individual log statements. I have reverted it and given as you suggested.